{
    "CVE_data_meta": {
        "ASSIGNER": "secalert@redhat.com",
        "ID": "CVE-2014-0161",
        "STATE": "PUBLIC"
    },
    "affects": {
        "vendor": {
            "vendor_data": [
                {
                    "vendor_name": "ovirt-engine-sdk-python",
                    "product": {
                        "product_data": [
                            {
                                "product_name": "ovirt-engine-sdk-python",
                                "version": {
                                    "version_data": [
                                        {
                                            "version_value": "before 3.4.0.7 and 3.5.0.4"
                                        }
                                    ]
                                }
                            }
                        ]
                    }
                }
            ]
        }
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
        "description_data": [
            {
                "lang": "eng",
                "value": "ovirt-engine-sdk-python before 3.4.0.7 and 3.5.0.4 does not verify that the hostname of the remote endpoint matches the Common Name (CN) or subjectAltName as specified by its x.509 certificate in a TLS/SSL session. This could allow man-in-the-middle attackers to spoof remote endpoints via an arbitrary valid certificate."
            }
        ]
    },
    "problemtype": {
        "problemtype_data": [
            {
                "description": [
                    {
                        "lang": "eng",
                        "value": "does not verify that the remote endpoint's hostname matches a domain name in the subject's CN or subjectAltName field of the x.509 certificate"
                    }
                ]
            }
        ]
    },
    "references": {
        "reference_data": [
            {
                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2014-0161",
                "refsource": "MISC",
                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2014-0161"
            },
            {
                "url": "https://access.redhat.com/security/cve/cve-2014-0161",
                "refsource": "MISC",
                "name": "https://access.redhat.com/security/cve/cve-2014-0161"
            }
        ]
    }
}